sample-client-app/sample-app-with-shared-lib/src/main/java/com/vmware/vipclient/sample/springboot/Application.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019-2020 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vipclient.sample.springboot;
+
+import java.util.Arrays;
+
+import com.vmware.vipclient.sample.Main;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class Application {
+
+	public static void main(String[] args) {
+		SpringApplication.run(Application.class, args);
+	}
+
+	@Bean
+	public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
```

Comment:
Why need a CommandLineRunner?

Reply:
It's just a way in Spring Boot to automatically run some logic after the application context is loaded. There are many ways to do it in Spring (https://reflectoring.io/spring-boot-execute-on-startup/), but we only need one here to run Main.main automatically which triggers all the sample code. Do you prefer any?